test/lint/README.md

```diff
@@ -19,6 +19,7 @@ maintained:
 * for `src/leveldb`: https://github.com/bitcoin-core/leveldb.git (branch bitcoin-fork)
 * for `src/univalue`: https://github.com/bitcoin-core/univalue.git (branch master)
 * for `src/crypto/ctaes`: https://github.com/bitcoin-core/ctaes.git (branch master)
+* for `src/crc32c`: https://github.com/google/crc32c.git (branch master)
```

Comment:
test: Add crc32c to subtree check linter 
 
 doesn't do https://github.com/bitcoin/bitcoin/pull/17398/commits/84ff1b2076ef91ce688930d0aa0a7f4078ef3e1d#diff-a12151b9941bfec9b94144ff38616fb4e3c081bc96a25591f966eef045eafeecR17
 
 Why?